cogs/csgo.py

```diff
@@ -82,6 +82,14 @@ async def pug(self, ctx: commands.Context, *args):
                 else:
                     raise commands.CommandError(message=f'Invalid Argument: `{arg}`')
 
+        if not self.pug.enabled:
+            if len(self.bot.queue_captains) > 0:
+                team1_captain_arg = self.bot.queue_captains[0]
+                self.bot.queue_captains.remove(team1_captain_arg)
```

Comment:
```suggestion
                team1_captain_arg = self.bot.queue_captains.pop(0)
```
Can be changed to this

Reply:
fixed in latest